google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -112,7 +112,7 @@ public static HttpTransportOptions getDefaultHttpTransportOptions() {
     return SCOPES;
   }
 
-  protected BigQueryRpc getBigQueryRpc() {
+  protected BigQueryRpc getBigQueryRpcV2() {
```

Comment:
What is the benefit of this rename? Are we planning on having `getBigQueryRpcV3` as well when it is available? How would we implement that?